### PR TITLE
allow non-admin users to PUT/POST to create resources

### PR DESCRIPTION
**Pull request by acoburn**
Resolves: https://jira.duraspace.org/browse/FCREPO-1809

### CODE REVIEW

File: src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java [hunk 1]
```diff
@@ -98,8 +99,20 @@ public void deleteRoles(final Node node) throws RepositoryException {
...
+    }
+
+    private FedoraResource locateResource(final Path path, final Session session) {
+
+        final Predicate<Path> exists = UncheckedPredicate.uncheck(x -> session.nodeExists(x.toString()));
+
+        if (exists.test(path) || path.isRoot()) {
+            LOGGER.debug("findRolesForPath: {}", path.getString());
+            return nodeService.find(session, path.toString());
+        } else {
```
Comment (awoods) [inline src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java]: No `else` necessary due to the `return` on line:111.

Comment (acoburn) [inline src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java]: you mean?

    if (condition) {
        return first;
    }
    return other;

I prefer my formulation, but I don't feel strongly about it.

Comment (awoods) [inline src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java]: That is what I mean... and I am looking out for @ajs6f here, as he seems to have a strong preference.

**Comment by acoburn** [comment]
@awoods the format of the conditional block has been changed.

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4/fcrepo-module-auth-webac/commit/26d1fccb6b6c2adeb8577b490b2fbabfa312e1ed

**Pull request closed without merging**